app/api/dao/section_dao.py

```diff
@@ -70,3 +70,7 @@ def delete_section_by_id(id:int) -> bool:
     @staticmethod
     def delete_section(section):
         return section.delete_from_db()
+
+    @staticmethod
+    def get_all_sections():
```

Comment:
Can you please name this with `find` prefix for consistency sake -> `find_all` or `find_all_sections` is good enough?